core/templates/components/entity-creation-services/exploration-creation.service.ts

```diff
@@ -39,7 +39,7 @@ angular.module('oppia').factory('ExplorationCreationService', [
     var explorationCreationInProgress = false;
 
     return {
-      createNewExploration: function() {
+      createNewExploration: function(params = {}) {
```

Comment:
Please add explicit name for the param here, maybe explorationTitle? and this should be a string instead of dict, you can construct a dict on line 51 using this value.

Reply:
Explaining latest change via comment:
In latest changes:--
I have pre-existing service named `exploration-creation.service` and create new function in that and used that function in `collection-node-creator.directive` to make a post request at `exploration/create_new`

I have passed title(string) as parameter in that function and create dict while making post request.